docs/articles/documentation/extending-testcafe/index.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: docs
+title: Extending TestCafe
+permalink: /documentation/extending-testcafe/
+---
+# Extending TestCafe
+
+TestCafe is a powerful framework, however you may need a specific functionality that TestCafe does not provide initially.
+To extend the TestCafe core functionality, you can use plug-ins.
+You can install existing plugins or develop custom plugins that will suit your needs.
+
+The following topics provide detailed information on the most widely used plugins and extensions.
```

Comment:
According to intro this section should describe plugins in details. However, it just describes how to develop plugins. And we don't have information about using them. We need either change title or premise. BTW, we don't have description on how to find plugins anywhere. E.g. it's not quite obvious that you should search npm for packages with name `testcafe-reporter-` to find reporter plugins.